File: module1/src/main/java/yxd/com/module1/Module1Activity.java
package yxd.com.module1;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;

@Route(path = "/test1/module1")
public class Module1Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_module1);
    }
}

File: module1/src/main/java/yxd/com/module1/Module1Activity.kt
package yxd.com.module1

import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.kotlinlib.activity.AbstractActivity

@Route(path = "/test1/module1")
class Module1Activity : AbstractActivity() {

    override fun init(bundle: Bundle?) {
        setContentView(R.layout.activity_module1)
    }

}
